test/js-api/wasm-module-builder.js

```diff
@@ -1018,13 +1016,13 @@ class WasmModuleBuilder {
               break;
             case kWasmF32:
               section.emit_u8(kExprF32Const);
-              f32_view[0] = global.init;
-              section.emit_bytes(f32_bytes_view);
+              data_view.setFloat32(0, global.init, true);
+              section.emit_bytes(byte_view.subarray(0, 4));
```

Comment:
Maybe this can be `section.emit_bytes(wasmF32Const(global.init))`, and similar below?

Reply:
`wasmF32Const` contains the `kExprF32Const` opcode, which is not wanted here.

Reply:
Oh, I see that it's indeed emitted right before :)
Yes, we could rewrite it. Let me do that.

Reply:
Done, also upstream (in v8): https://crrev.com/c/2257229